plugins/modules/system/ufw.py

```diff
@@ -285,7 +285,7 @@
 - name: Deny forwarded/routed traffic from subnet 1.2.3.0/24 to subnet 4.5.6.0/24
   community.general.ufw:
     rule: deny
-    route: yes
+    route: true
     src: 1.2.3.0/24
```

Comment:
```suggestion
    src: 192.0.2.0/24
```

Reply:
strictly speaking, we shouldn't use real IP addresses. 1.2.3.0 is actually owned by someone ;-) The two replacements here are documentation reserved IP addresses per https://www.rfc-editor.org/rfc/rfc5737

Reply:
I'm sure there are more IP addresses all over the collection's examples... If someone is sufficiently bored, they can go searching and replacing ;-) (and I don't want to imply that this hypothetical someone is you :) ) I've added these two changes to the PR.